docs/flows/crypto-offramp.mdx

```diff
@@ -7,4 +7,586 @@ import TabItem from "@theme/TabItem";
 
 # Crypto off-ramp
 
-The crypto off-ramp flow allows a user to sell cryptocurrency and convert it into fiat, which can then be transferred to their linked debit cards.
\ No newline at end of file
+The crypto off-ramp flow allows a user to sell cryptocurrency and convert it into fiat, which can then be transferred to their linked debit cards.
+
+## Bootstrap token payload
+
+<Tabs>
+  <TabItem label="Schema" value="schema" default>
+
+- `partner` (_optional_): Partner object.
+  - `continueUrl` (_optional_): URL to redirect the user to after the order is placed.
```

Comment:
URL to redirect the user to after the order is placed and the funds were received.